### PR TITLE
Chapter 3: Mostly static pages

### PR DESCRIPTION
**Pull request by rashidfirhat98**

### CODE REVIEW

**Comment by rashidfirhat98** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Sample App"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
+  end
```
Comment (saobv-2223) [inline app/helpers/application_helper.rb]:
```suggestion
  def full_title page_title
    base_title = "Sample App"
    page_title.blank? ? base_title : "#{page_title} | #{base_title}"
  end
```

File: app/views/static_pages/help.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+    Get help on the Ruby on Rails Tutorial at the
+    <%= link_to "Rails Tutorial help page", Settings.links.rails_tutorial.help_page %>.
+    To get help on this sample app, see the
+    <%= link_to "<em>Ruby on Rails Tutorial</em> book", Settings.links.rails_tutorial.book_page %>.  
+</p>
```
Comment (saobv-2223) [inline app/views/static_pages/help.html.erb]:
```suggestion
<p>
  Get help on the Ruby on Rails Tutorial at the
  <%= link_to "Rails Tutorial help page", Settings.links.rails_tutorial.help_page %>.
  To get help on this sample app, see the
  <%= link_to "<em>Ruby on Rails Tutorial</em> book", Settings.links.rails_tutorial.book_page %>.  
</p>

```

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<%= provide :title, "Home" %>
+
+<h1>Sample App</h1>
+<p>
+    This is the home page for the
+    <%= link_to "Ruby on Rails Tutorial", Settings.links.rails_tutorial.home_page %>
+    sample application.
+</p>
```
Comment (saobv-2223) [inline app/views/static_pages/home.html.erb]:
```suggestion
<p>
  This is the home page for the
  <%= link_to "Ruby on Rails Tutorial", Settings.links.rails_tutorial.home_page %>
  sample application.
</p>
```

File: config/database.yml [hunk 4]
```diff
@@ -1,25 +1,19 @@
...
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  adapter: mysql2
```
Comment (saobv-2223) [inline config/database.yml]: image

File: config/routes.rb [hunk 5]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root 'static_pages#home'
+  get 'static_pages/help'
```
Comment (saobv-2223) [inline config/routes.rb]: using " instead of '

File: config/settings.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+links:
+  rails_tutorial:
+    help_page: http://www.railstutorial.org/help
+    book_page: https://www.railstutorial.org/book
+    home_page: http://www.railstutorial.org/
+
```
Comment (saobv-2223) [inline config/settings.yml]:
```suggestion
links:
  rails_tutorial:
    help_page: http://www.railstutorial.org/help
    book_page: https://www.railstutorial.org/book
    home_page: http://www.railstutorial.org/
```

**Comment by rashidfirhat98** [comment]
ready

**Comment by ngocvt-0484** [comment]
merge please

**Comment by vanvtt-0952** [comment]
merged
Please fix comment, then merge the pull request yourself

**Pull request merged**